docs/content/en/highlights/2021-07-14-0-15-upgrade-guide.md

```diff
@@ -0,0 +1,125 @@
+---
+date: "2021-07-14"
+title: "0.15 Upgrade Guide"
+description: "An upgrade guide that addresses breaking changes in 0.15.0"
+authors: ["jszwedko"]
+pr_numbers: []
+release: "0.15.0"
+hide_on_release_notes: false
+badges:
+  type: breaking change
+---
+
+Vector's 0.15.0 release includes a few minor breaking changes:
+
+1. [Dropping support for Kubernetes 1.14.](#first)
+2. [The `sample` transform now takes VRL conditions by default.](#second)
+3. [The `remap` condition type was renamed to `vrl`.](#third)
+
+We cover each below to help you upgrade quickly:
+
+## Upgrade Guide
+
+### Dropping support for Kubernetes 1.14<a name="first"></a>
+
+We've dropped support for Kubernetes version 1.14 as it is no longer a supported Kubernetes version nor do major cloud
+providers support it.
+
+This version of Kubernetes is likely to still work with Vector, but we will no longer be testing against it and cannot
+guarantee future versions of Vector will be compatible.
+
+In general, we will aim to match [Kuberentes' supported releases][kubernetes_releases] and along with the versions
+supported by major cloud providers.
+
+### The `sample` transform now takes VRL conditions by default<a name="second"></a>
+
+In version 0.12.0 we switched to using [VRL][vrl] as the default condition type for our transforms that take conditions (like
```

Comment:
You can just use `[VRL]` for links like this

Reply:
Aha nice! The old site wouldn't allow that iirc.